src/main/resources/network/brightspots/rcv/config_file_documentation.txt

```diff
@@ -113,10 +113,60 @@ Config file must be valid JSON format. Examples can be found in the test_data fo
     set of configuration parameters that specify the tabulation rules
     the "rules" section contains the following parameters:
 
+      "randomSeed" required when tiebreakMode is random, previousRoundCountsThenRandom, or generatePermutation
+        the integer seed for the application's pseudorandom number generator
+        value: [0..2147483647]
+        if not supplied: null
+
       "tiebreakMode" required
-        how the program should decide which candidate to eliminate when multiple candidates are tied for last place among continuing candidates in a given round
+        how the program should decide which candidate to eliminate when multiple candidates are tied for last place
+        or which candidate to elect first when:
+          electionWinnerMode is set to multiSeatAllowOnlyOneWinnerPerRound and
+          multiple candidates exceed the winning threshold and
+          also tie for the highest vote total in the same round
         value: "random" | "interactive" | "previousRoundCountsThenRandom" | "previousRoundCountsThenInteractive" | "usePermutationInConfig" | "generatePermutation"
 
+        we use Java 12 java.util.random in our tiebreak implementations as follows:
+
+        "random"
+          on config load a java.util.Random object is created using the randomSeed value specified in the input config file:
+          random = new Random(randomSeed);
+          during tabulation, in the event of a tie at the end of a round:
+          the list of tied candidates is sorted alphabetically
+          a randomDouble is created using the random object:
+            double randomDouble = random.nextDouble();
+            the randomDouble is mapped to one of the tied candidates in the list:
+            int randomCandidateIndex = (int) Math.floor(randomDouble * (double) tiedCandidates.size());
+          the selected candidate will be the winner or loser for that round
+
+        "interactive":
+          the user is presented with a list of tied candidates
+          the user will input their selection manually
+
+        "previousRoundCountsThenRandom"
+          the tied candidate with the highest vote total in the previous round is selected
+          if there is a tie for the vote count in the previous round as well, a candidate is selected from the still tying candidates as described under tiebreakMode "random"
```

Comment:
I actually haven't implemented this part properly. If it has to fall back to random or interactive, it picks from the full set of tied candidates, not just the ones who were still alive at the end of the previousRoundCounts logic. I'll send a PR to fix this shortly.